suites/quincy/cephadm/tier-0_dmfg.yaml

```diff
@@ -0,0 +1,119 @@
+#======================================6=========================================================
```

Comment:
Do we need run these tests as part of build sanity check?

Reply:
As of now we will be running this along with other suites, if there is any changes required we will take this as an improvement